core/shared/src/main/scala/fs2/concurrent/Queue.scala

```diff
@@ -143,11 +143,11 @@ object Queue {
 
   /** Creates a queue with no size bound. */
   def unbounded[F[_], A](implicit F: Concurrent[F]): F[Queue[F, A]] =
-    forStrategy(Strategy.lifo[A])
+    forStrategy(Strategy.fifo[A])
 
   /** Creates an unbounded queue that distributed always at max `fairSize` elements to any subscriber. */
   def fairUnbounded[F[_], A](fairSize: Int)(implicit F: Concurrent[F]): F[Queue[F, A]] =
-    forStrategy(Strategy.lifo[A].transformSelector[Int]((sz, _) => sz.min(fairSize)))
+    forStrategy(Strategy.fifo[A].transformSelector[Int]((sz, _) => sz.min(fairSize)))
 
   /** Creates a queue with the specified size bound. */
   def bounded[F[_], A](maxSize: Int)(implicit F: Concurrent[F]): F[Queue[F, A]] =
```

Comment:
Looks like `bounded`, `boundedNoneTerminated`, and `fairBounded` need to be updated too.

Reply:
You're right! That should explain the failed test.